error.go

```diff
@@ -265,6 +299,14 @@ func (e *ErrorResponse) Token() string { return e.ID }
 // Merge returns the updated error. This is useful in case the error was initially nil in
 // which case other is returned.
 func MergeErrors(err, other error) error {
+	// If either err is mergable, merge them.
```

Comment:
Maybe it would be better if `ErrorResponse` didn't implement ServiceMergableError. Removes the possibility of the old merging logic to be used.